.circleci/config.yml

```diff
@@ -1035,7 +1035,7 @@ jobs:
       - image: cimg/node:12.16
     environment:
       PUBLIC_URL: /0
-      SERVER_ADDRESS: http://localhost:8080
+      SERVER_ADDRESS: https://detai.ml/proxy
```

Comment:
**question:** I'm a bit clueless here, but where is this URL from?

Reply:
**suggestion:** would it help to have a comment above this explaining that it allows https connectivity and that it points to `latest-master.determined.at:8080`?